docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -44,22 +44,22 @@ The following example uses a regular expression to verify that a string is in va
   
 |Pattern|Description|  
 |-------------|-----------------|  
 |`^`|Begin the match at the start of the string.|  
 |`(?(")`|Determine whether the first character is a quotation mark. `(?(")` is the beginning of an alternation construct.|  
-|`(?("")("".+?(?<!\\)""@)`|If the first character is a quotation mark, match a beginning quotation mark followed by at least one occurrence of any character, followed by an ending quotation mark. The ending quotation mark must not be preceded by a backslash character (\\). `(?<!` is the beginning of a zero-width negative lookbehind assertion. The string should conclude with an at sign (@).|  
+|`(?(")(".+?(?<!\\)"@)`|If the first character is a quotation mark, match a beginning quotation mark followed by at least one occurrence of any character, followed by an ending quotation mark. The ending quotation mark must not be preceded by a backslash character (\\). `(?<!` is the beginning of a zero-width negative lookbehind assertion. The string should conclude with an at sign (@).|  
 |<code>&#124;(([0-9a-z]</code>|If the first character is not a quotation mark, match any alphabetic character from a to z or A to Z (the comparison is case insensitive), or any numeric character from 0 to 9.|  
-|`(\.(?!\.))`|If the next character is a period, match it. If it is not a period, look ahead to the next character and continue the match. `(?!\.)` is a zero-width negative lookahead assertion that prevents two consecutive periods from appearing in the local part of an email address.|  
-|<code>&#124;[-!#\$%&'\*\+/=\?\^\`{}&#124;~\w]</code>|If the next character is not a period, match any word character or one of the following characters: -!#$%'*+=?^\`{}&#124;~.|  
-|<code>((\.(?!\.))&#124;[-!#\$%'\*\+/=\?\^\`{}&#124;~\w])*</code>|Match the alternation pattern (a period followed by a non-period, or one of a number of characters) zero or more times.|  
+|`((\.(?!\.))`|If the next character is a period, match it. If it is not a period, look ahead to the next character and continue the match. `(?!\.)` is a zero-width negative lookahead assertion that prevents two consecutive periods from appearing in the local part of an email address.|  
+|<code>&#124;[-!#\\$%&'\\*\\+/=\?\^\`\{\}\&#124;~\w]</code>|If the next character is not a period, match any word character or one of the following characters: -!#$%'*+=?^\`{}&#124;~.|  
+|<code>((\\.(?!\\.))&#124;[-!#\\$%'\\*\\+/=\?\^\`\{\}\&#124;~\w])*</code>|Match the alternation pattern (a period followed by a non-period, or one of a number of characters) zero or more times.|  
```

Comment:
This is still not 100%. Let me research how we can achieve the desired result and get rid of these of the \<code> elements here.

![image](https://user-images.githubusercontent.com/12971179/66356408-bc110200-e91f-11e9-8fb8-31dd1dc595f4.png)